internal/controller/auth.go

```diff
@@ -18,7 +18,7 @@ func AuthEndpoints(service service.AuthService) rest.Routes {
 	var out rest.Routes
 
 	authHandler := fromAuthServiceAwareHttpHandler(authUser, service)
-	auth := rest.NewRoute(http.MethodGet, false, "/users/auth", authHandler)
+	auth := rest.NewRoute(http.MethodGet, false, "/auth", authHandler)
```

Comment:
The `users` part comes from the `Prefix` now: no need to specify it here. This is also cleaner.